pkg/image/nbdkit.go

```diff
@@ -0,0 +1,159 @@
+package image
+
+import (
+	"fmt"
+	"github.com/pkg/errors"
+	"github.com/prometheus/client_golang/prometheus"
+	"k8s.io/klog/v2"
+	"kubevirt.io/containerized-data-importer/pkg/common"
+	"kubevirt.io/containerized-data-importer/pkg/system"
+	"kubevirt.io/containerized-data-importer/pkg/util"
+	"net/url"
+	"strings"
+)
+
+var (
+	nbdkitExecFunction = system.ExecWithLimits
+)
+
+// NbdkitPlugin represents a plugin for nbdkit
+type NbdkitPlugin string
+
+// NbdkitFilter represents s filter for nbdkit
+type NbdkitFilter string
+
+// Nbdkit plugins
+const (
+	NbdkitCurlPlugin NbdkitPlugin = "curl"
+)
+
+// Nbdkit filters
+const (
+	NbdkitXzFilter   NbdkitFilter = "xz"
+	NbdkitTarFilter  NbdkitFilter = "tar"
+	NbdkitGzipFilter NbdkitFilter = "gzip"
+)
+
+func (p NbdkitPlugin) String() string {
+	return string(p)
+}
+func (f NbdkitFilter) String() string {
+	return string(f)
+}
+
+func init() {
```

Comment:
Don't need this, since the qemu.go in the same package already calls it.

Reply:
ok thanks :)